value.go

```diff
@@ -61,11 +61,57 @@ func (v *valueBase) aValue() {}
 
 type valueString struct {
 	valueBase
-	value string
+	// We use rune slices instead of strings for quick indexing
+	value []rune
+}
+
+func (s *valueString) index(e *evaluator, index int) (value, error) {
+	runeIndex := 0
+	for _, r := range s.value {
```

Comment:
Why not check if `index` is in the correct bounds, and if so, just index directly into `s.value`?

Reply:
Ah, when I was changing representation, I forgot to change that. Thanks (this will be changed in the other PR).

Reply:
done (in https://github.com/google/go-jsonnet/pull/34)